src/overlays/actors/ovl_Door_Shutter/z_door_shutter.h

```diff
@@ -46,15 +46,15 @@ typedef void (*DoorShutterActionFunc)(struct DoorShutter*, PlayState*);
 typedef struct DoorShutter {
     /* 0x0000 */ DynaPolyActor dyna;
     /* 0x0164 */ s16 unk_164;
-    /* 0x0166 */ s16 unk_166;
-    /* 0x0168 */ s16 unk_168;
+    /* 0x0166 */ s16 jabuDoorClosed; // Ranges from 0 (open) to 100 (closed)
```

Comment:
would the same apply to `f32 barsClosed` (0.0f to 1.0f) ?

it's indeed not a bool, but close to it, it stays 0 when the door is open and stays 100 when the door is closed, so it's both a state and a timer

I'm okay with `jabuDoorClosedPercentage` but that wouldn't work for `barsClosed`

Reply:
yeah it would be good for them to be named similarly. 
would "amount" work? closed amount, and the comment explains the range for both since theyre different